src/pages/home/components/Sidebar/Sidebar.spec.tsx

```diff
@@ -1,18 +1,208 @@
+          id: 'DISCORD_REELS_POST_ID',
+          name: 'TestePost',
+          previewComponent: 'Teste',
+          validators: {
+            text: {
+              maxLength: 3,
+            },
+          },
+          widgets: [
+            {
+              component: 'Teste',
+              icon: 'Teste',
+              name: 'Teste',
+            },
+          ],
+        },
+        {
+          id: 'DISCORD_REELS_POSTMODE_ID',
+          name: 'TesteReels',
+          previewComponent: 'Teste',
+          validators: {
+            text: {
+              maxLength: 3,
+            },
+          },
+          widgets: [
+            {
+              component: 'Teste',
+              icon: 'Teste',
+              name: 'Teste',
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  [
+    'TWITTER_EXAMPLE_ID',
+    {
+      icon: 'Icon',
+      id: 'TWITTER_EXAMPLE_ID',
+      name: 'Twitter',
+      postModes: [
+        {
+          id: 'TWITTER_THREAD_POST_ID',
+          name: 'TestePost',
+          previewComponent: 'Teste',
+          validators: {
+            text: {
+              maxLength: 3,
+            },
+          },
+          widgets: [
+            {
+              component: 'Teste',
+              icon: 'Teste',
+              name: 'Teste',
+            },
+          ],
+        },
+        {
+          id: 'TWITTER_THREAD_POSTMODE_ID',
+          name: 'TesteThread',
+          previewComponent: 'Teste',
+          validators: {
+            text: {
+              maxLength: 3,
+            },
+          },
+          widgets: [
+            {
+              component: 'Teste',
+              icon: 'Teste',
+              name: 'Teste',
+            },
+          ],
+        },
+      ],
+    },
+  ],
+]);
+
 // TODO: revisitar esse teste pois está falhando
-describe.skip('Sidebar', () => {
+describe('Sidebar component', () => {
   it('renders correctly', () => {
+    vi.spyOn(useSocialMediaStoreModule, 'useSocialMediaStore').mockReturnValue({
+      accounts: mockAccounts,
+      socialMedias: mockSocialMedias,
+    });
+
     render(<Sidebar />);
 
     const sideBarComponentEvidence = screen.getByText(/Select Social Media/);
 
     expect(sideBarComponentEvidence).toBeInTheDocument();
   });
 
+  it('renders accounts correctly', async () => {
+    vi.spyOn(useSocialMediaStoreModule, 'useSocialMediaStore').mockReturnValue({
+      accounts: mockAccounts,
+      socialMedias: mockSocialMedias,
+    });
+
+    render(<Sidebar />);
+
+    const [account] = mockAccounts;
+    const socialMedia = mockSocialMedias.get(account.socialMediaId);
+
+    const accordionEvidence = screen.getByText(socialMedia?.name as string);
+    await userEvent.click(accordionEvidence);
+
+    const accountEvidence = screen.getByText(account.userName);
+
+    expect(accordionEvidence).toBeInTheDocument();
+    expect(accountEvidence).toBeInTheDocument();
+  });
+
+  it('renders when accounts is empty correctly', () => {
```

Comment:
no caso aqui, `dont renders accounts when accounts store is empty`

Na sua descrição precisa explicar o motivo de ser correctly, correclty em si não tem significado, por siso precisamos deixar claro oq o teste é